src/pages/my-claims/roles.tsx

```diff
@@ -0,0 +1,50 @@
+import ExternalLink from '../../components/external-link';
+import Api3Icon from '../../components/icons/api3-icon';
+import ExternalLinkIcon from '../../components/icons/external-link-icon';
+import KlerosIcon from '../../components/icons/kleros-icon';
+import { Tooltip } from '../../components/tooltip';
+import { images } from '../../utils';
+import styles from './roles.module.scss';
+
+export default function Roles() {
+  return (
+    <section className={styles.rolesSection}>
+      <h6>
+        Roles
+        <Tooltip
+          id="roles-tooltip"
+          overlay={
+            <div className={styles.tooltip}>
+              <p>
+                The <b>mediator, API3 Mediators,</b> is responsible for deciding the initial outcome of claims and
```

Comment:
This text doesn't read nice to me:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/22679154/194744785-b29f1321-ecd1-417e-bfc9-2e2dc7c6d3b2.png">

I'd maybe expect the "API3 Mediators" and "Kleros" to be in brackets. WDYT?

Reply:
Yeah I agree 👍